src/sql/src/plan/statement/ddl.rs

```diff
@@ -1027,18 +1027,25 @@ pub fn plan_create_source(
         )
     }
 
-    Ok(Plan::CreateSource {
+    Ok(Plan::CreateSource(CreateSourcePlan {
         name,
         source,
         if_not_exists,
         materialized,
-    })
+    }))
 }
 
 pub fn plan_create_sources(
-    _scx: &StatementContext,
-    _stmt: CreateSourcesStatement<Raw>,
+    scx: &StatementContext,
+    stmt: CreateSourcesStatement<Raw>,
 ) -> Result<Plan, anyhow::Error> {
+    scx.require_experimental_mode("Postgres Sources")?;
```

Comment:
I have to say that I am pretty lost here! I'm not sure why `plan_create_sources` gets winnowed down to only caring about Posgres sources, so I might not be the right person to review.

Reply:
You're totally right that `CREATE SOURCES` will only work with Postgres sources at the moment. [The idea](https://github.com/MaterializeInc/materialize/pull/6299#issuecomment-812102095) was that users would be able to provide the connection information for Postgres once, while specifying multiple upstream tables. I'm not sure if there's any plan to expand it to other source types in the near-term or long-term future.